Widgets/AIChatControlWidget.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  AIChatControlWidget.swift
+//  DuckDuckGo
+//
+//  Copyright © 2025 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import WidgetKit
+import SwiftUI
+import AppIntents
+
+/// Will be added once we have the correct assets for it
```

Comment:
Waiting on https://app.asana.com/0/72649045549333/1209211536736923/f
But you can test the functionality by adding the `AIChatControlWidget` on Widgets.swift